woocommerce/woocommerce-gutenberg-products-block
# Add stories for Button component #5250

This PR adds 3 stories for the Button component:
- Default
- Disabled
- With Spinner

It also adds Storybook controls to change dynamically the props of the Button component

@sunyatasattva in the original issue (#5250 ), you have suggested the following names for the stories: Disallow add to cart, Adding to cart state, Added to cart state.

We have decided to keep the names a bit more generic: Default, Disabled, With Spinner as the Button component is used in multiple contexts. What do you think?


Fixes #5250 

### Screenshots

<img width="1346" alt="Screenshot_02_02_2022__15_40" src="https://user-images.githubusercontent.com/905781/152175443-67ade385-47ed-4c86-8400-f9963a5a4f07.png">

### Testing

### Manual Testing
How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [Button stories](http://localhost:6006/?path=/story/woocommerce-blocks-base-components-button--default). You should see one default story, and two additional stories with the appropriate controls.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).


Co-authored-by: gigitux <gigitux@gmail.com>
